bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html

```diff
@@ -254,52 +254,101 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-home-join-the-movement') }}</h2>
         ) }}
       {% endif %}
 
-      {{ gallery_tile(
-        class='m24-l-grid-quarter',
-        image=picture(
-          url="img/home/2024/ai-gallery/builders-mobile-720.jpg",
-          sources=[
-            {
-              'media': '(max-width: 1311px)',
-              "srcset" : {
-                "img/home/2024/ai-gallery/builders-mobile-375.jpg": "375w",
-                "img/home/2024/ai-gallery/builders-mobile-720.jpg": "720w",
-                "img/home/2024/ai-gallery/builders-mobile-976.jpg": "976w",
-                "img/home/2024/ai-gallery/builders-mobile-1232.jpg": "1232w",
-                "img/home/2024/ai-gallery/builders-mobile-1450.jpg": "1450w",
+      {% if ftl_has_messages('m24-home-mozfest-join-us', 'm24-home-mozfest-the-future', 'm24-home-mozfest-get-your' ) %}
```

Comment:
Given limited timing, is it better to show without condition and remove builders block completely?